### PR TITLE
Removed Rails dependency

### PR DESCRIPTION
**Pull request by tachyons**
1. Fixed deprecated methods from test case
2. Removed Rails dependency , ( a synatra app with activerecord can also use this gem)
3. Improved test speed ( ~1 sec to .03 sec)
4. Gem home page fix

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
   - gem update --system
-install: "bundle install"
+install:
+  - "gem install bundler"
+  - "bundle install --jobs=3 --retry=3"
 script: "cd spec/dummy/; bundle exec rake db:create; bundle exec rake db:migrate; bundle exec rspec spec/"
 branches:
   only:
     - master
+    - best_practices
```
Comment (Charizard) [inline .travis.yml]: @tachyons I believe this is not needed.

Comment (tachyons) [inline .travis.yml]: @Charizad : Yeah , I added this for testing my fork . I can remove this now :-)

**Comment by Charizard** [comment]
@tachyons Thank you.

@sanandnarayan The PR looks good. Can you merge this?

**Comment by nithinkrishna** [comment]
@Charizard What picture is that? xD

**Pull request merged**